### PR TITLE
Add 46Elks channel

### PR DESCRIPTION
**Pull request by larsemil**

### CODE REVIEW

**Comment by atymic** [comment]
Hi!

Are you still open to creating and maintaining this channel?

If so, could you update it to support laravel 5.5-6.0 please?

Thanks!

**Comment by larsemil** [comment]
I will look into it.

File: src/FortySixElksMedia.php [hunk 1]
```diff
@@ -44,7 +44,9 @@ public function __construct()
     {
         $this->username = config('services.46elks.username');
         $this->password = config('services.46elks.password');
-
+        if (!$this->username || !$this->password) {
+            throw MissingConfigNotification::missingConfig();
+        }
         if ($this->username && $this->password) {
```
Comment (atymic) [inline src/FortySixElksMedia.php]: you can remove this `if` statement now

File: tests/ExampleTest.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace NotificationChannels\FortySixElks\Test;
+
+class ExampleTest extends \PHPUnit_Framework_TestCase
+{
+    /** @test */
+    public function true_is_true()
```
[the remark this answers is not in the document]
Comment (larsemil) [reply tests/ExampleTest.php]: I will look into it. But for now

[the remark this answers is not in the document]
Comment (atymic) [reply tests/ExampleTest.php]: Thanks :)

[the remark this answers is not in the document]
Comment (larsemil) [reply tests/ExampleTest.php]: Some simple tests added

**Comment by atymic** [comment]
Could you just add another quick test to cover the actual notify logic? Similar to this:
https://github.com/laravel-notification-channels/new-channels/pull/35/files#diff-6e36f1a1becd7049f27f18c6212bced0

I'll create a repo now :)

**Comment by atymic** [comment]
See https://github.com/laravel-notification-channels/46elks/issues/1

**Pull request closed without merging**